modules/fileout_netcdf/FONcBaseType.h

```diff
@@ -66,6 +66,9 @@ class FONcBaseType: public BESObj {
     bool d_defined = false;
     std::string d_ncVersion;
     std::string d_nc4_datamodel;
+    // FIXME Tried setting this true by default; that should have changed some test behavior
```

Comment:
As I mentioned, the d_is_dap4 is set by transform_dap4() in FONCtranform.cc when DAP4 request is asked.

Reply:
OK. I found four places where d_is_dap4 is set by calling FONcBaseType::convert() from within transform_dap4() helper functions and I added comments so we can find those in the future.